internal/util/util.go

```diff
@@ -158,6 +157,40 @@ type KernelVersion struct {
 	Backport     bool   // backports have a fixed version/patchlevel/sublevel
 }
 
+// Parses a kernel release version string into:
+//   version, patch version, sub version and extra version
+func ParseKernelRelease(release string) (int, int , int, int, error) {
+	version := 0
+	patchlevel := 0
+	sublevel := 0
+	extraversion := 0
+
+	extra := ""
+	n, err := fmt.Sscanf(release, "%d.%d%s", &version, &patchlevel, &extra)
+	if n < 2 && err != nil {
+		err = fmt.Errorf("failed to parse version and patchlevel from %s: %v", release, err)
```

Comment:
```suggestion
err = fmt.Errorf("failed to parse version and patchlevel from %s: %w", release, err)
```

Reply:
use `%w` in fmt.Errorf for  error in other places also

Reply:
Done, thank you.